src/client/renderer/r_bsp_draw.c

```diff
@@ -452,9 +450,7 @@ static void R_DrawBspInlineModelAlphaTestDrawElements(const r_view_t *view,
 			continue;
 		}
 		
-		R_TIMER_WRAP(va("AlphaTestDrawElem: %s", draw->material->media.name),
```

Comment:
I took the timers stuff out @Paril 😬 It was driving me nuts with X-code's indentation and syntax highlighting being completely hosed, plus breakpoints not going where I put them. I removed the timers in a separate commit so if we want to re-add it should be relatively easy.